src/aws/osml/model_runner/database/region_request_table.py

```diff
@@ -152,3 +205,32 @@ def get_region_request(self, region_id: str, image_id: str) -> Optional[RegionRe
         except Exception as err:
             logger.warning(GetRegionRequestItemException(f"Failed to get RegionRequestItem! {err}"))
             return None
+
+    def add_tile(self, image_id: str, region_id: str, tile: ImageRegion, state: str) -> dict[str, Any]:
```

Comment:
Should this state really be a `str` value or should it be a more restrictive enum to ensure it is only ever "succeeded" or "failed"? This only really caught my eye because there isn't any validation in this function and the value is used directly to create the name of the property in DDB.

Reply:
Added enum value to typing